James issue#53

j2quik opened the pull request:
Forgot to add the test cases with the original pull request associated with Issue#28.

j2quik committed:
Added test cases for moon.py

Forgot to add the test cases with the original pull request associated with Issue#28.

JosephBehr reviewed (changes requested):
Good tests added to the code, did you make sure they pass with pytest? Also, remember to include reference to issues, @returns, @param[in], and @param[out] in front of relevant doxygen comments for literate programming and automatic documentation.

j2quik committed:
Updated test case for moon.py

This file only has test cases for moon.py. Added reference to Isssue#53 and relevant Doxygen comments for literate programming and automatic documentation.

j2quik commented:
I did run the test and it passed. I only included the test for moon.py in the updated file. Once approved, I will merge into test.py. I made the suggested improvements.

j2quik committed:
Updated test.py

Integrated tests for moon.py into test.py

JosephBehr reviewed (approved):
Thanks for making the changes, good to merge
